src/functions/node_helper.php

```diff
@@ -35,7 +35,7 @@ function dump_node($node, int $depth = 2): void
     /**
      * @param Node|Node[] $node
      */
-    function print_node($node): void
+    function print_node(Node | array $node): void
```

Comment:
I think this broke CI on downgrade:
https://github.com/rectorphp/rector-src/runs/2778581424

It's missed for some readon

![image](https://user-images.githubusercontent.com/924196/121268547-7a87e580-c8be-11eb-9964-0610e2c5b731.png)

Reply:
resolved at https://github.com/rectorphp/rector-src/pull/185